contract/package.json

```diff
@@ -9,7 +9,7 @@
         "compile": "npx hardhat compile",
         "multisig": "npx hardhat deploy --tags Multisig",
         "deployer": "npx hardhat deploy --tags Deployer",
-        "dns-sample": "npx hardhat deploy --network local --tags DNSSample",
+        "dns-sample": "npx hardhat deploy --network local --tags 'Deployer,DNSSample'",
```

Comment:
should remove Deployer since you use environment variable to configure contract addresses, so Deployer stuff is assumed to be in-existence already, not to be deployed again

Reply:
The goal of this command is to deploy contracts and dns records for local testing the current workflow used is documented as follows

```
# start local ganache in window 1
cd env
./ganache-new.sh

# deploy the contracts in window 2
cd contract
yarn install
cp .env.sample .env
yarn dns-sample
```

We do the deployer once initially and then update the environment variables, which are consistent (the same addresses) on a new ganache instance.
We then run this command (not running the deployer, updating config and then running DNSSample"

Other approaches would be to remove this and then run
```
yarn deployer
yarn dns-sample
```

imho: this is the cleaner solution

Reply:
It should not assume local environment deploys to particular addresses. The .sample file should be taken as a suggestion or example. We shouldn't expect developers to copy it in entirety. Two lines of yarn executions look better. The addresses can be left auto-detected if environment variable is empty